### PR TITLE
[zh-cn] sync kubelet object environment-variable-expose-pod-information

### PR DESCRIPTION
**Pull request by asa3311**
sync kubelet object environment-variable-expose-pod-information

### CODE REVIEW

File: content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md [hunk 1]
```diff
@@ -34,6 +39,11 @@ downward API.
 
 这两种呈现 Pod 和容器字段的方式统称为 downward API。
 
+由于服务是 Kubernetes 管理的容器化应用之间的主要通信模式，因此在运行时能发现这些服务是很有帮助的。
+
+[这里](/zh-cn/docs/tutorials/services/connect-applications-service/#accessing-the-service)
```
Comment (my-git9) [inline content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md:44]:
```suggestion
在[这里](/zh-cn/docs/tutorials/services/connect-applications-service/#accessing-the-service)
```

File: content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md [hunk 2]
```diff
@@ -34,6 +39,11 @@ downward API.
 
 这两种呈现 Pod 和容器字段的方式统称为 downward API。
 
+由于服务是 Kubernetes 管理的容器化应用之间的主要通信模式，因此在运行时能发现这些服务是很有帮助的。
```
Comment (my-git9) [inline content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md:42]: 服务  --> Service

File: content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md [hunk 3]
```diff
@@ -34,6 +39,11 @@ downward API.
...
+
+[这里](/zh-cn/docs/tutorials/services/connect-applications-service/#accessing-the-service)
+阅读更多关于访问服务的信息。
```
Comment (my-git9) [inline content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md:45]: 服务 --> Service

**Review by windsonsea — COMMENTED** [review]
/approve

File: content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md [hunk 4]
```diff
@@ -34,6 +39,11 @@ downward API.
 
 这两种呈现 Pod 和容器字段的方式统称为 downward API。
 
+由于 Service 是 Kubernetes 管理的容器化应用之间的主要通信模式，因此在运行时能发现这些服务是很有帮助的。
```
Comment (my-git9) [inline content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md:42]:
```suggestion
Service 是 Kubernetes 管理的容器化应用之间的主要通信模式，因此在运行时能发现这些 Service 是很有帮助的。
```

**Pull request merged**